usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -75,14 +75,16 @@ SELECT
   UTM.pop_county_name,
   UTM.pop_zip5,
   UTM.pop_congressional_code,
+  TRIM(TRAILING FROM COALESCE(FPDS.place_of_perform_city_name, FABS.place_of_performance_city)) AS pop_city_name,
 
   UTM.recipient_location_country_code,
   UTM.recipient_location_country_name,
   UTM.recipient_location_state_code,
   UTM.recipient_location_county_code,
   UTM.recipient_location_county_name,
   UTM.recipient_location_zip5,
-  UTM.recipient_location_congressional_code
+  UTM.recipient_location_congressional_code,
+  TRIM(TRAILING FROM COALESCE(FPDS.legal_entity_city_name, FABS.legal_entity_city_name)) AS recipient_location_city_name
```

Comment:
Why just trailing?  Will leading whitespace not mess things up?

Reply:
I didn't notice any leading whitespace